conda-recipes/iqsharp/build.ps1

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env pwsh
+param(
+    [string]
+    $Configuration = "Release",
+
+    [string]
+    $DotNetPath = $null
+);
+
+# If we weren't given a path, find dotnet now.
+if ($null -eq $DotNetPath) {
+    $DotNetPath = (Get-Command dotnet).Source;
+    if ($DotNetPath -eq $null) {
+        Write-Error "Could not find .NET Core SDK. This should never happen."
+        exit -1;
+    }
+}
+
+# The user may not have run .NET Core SDK before, so we disable first-time
+# experience to avoid capturing the NuGet cache.
+$Env:DOTNET_SKIP_FIRST_TIME_EXPERIENCE = "true";
+$Env:NUGET_XMLDOC_MODE = "skip";
+
+# On Windows PowerShell, we don't have the nice $IsWindows / $IsLinux / $IsMacOS
+# variables. Since Windows PowerShell is the only PowerShell variant that
+# has "Desktop" as its PSEdition, we can check for that and create the
+# $IsWindows variable if need be.
+if ($PSVersionTable.PSEdition -eq "Desktop") {
+    $IsWindows = $true;
+}
+
+if ($IsWindows) {
+    # NOTE: Building this package is ★only★ supported for Windows 10.
+    $RuntimeID = "win10-x$Env:ARCH";
+} elseif ($IsLinux) {
+    $RuntimeID = "linux-x$Env:ARCH";
+} elseif ($IsMacOS) {
+    $RuntimeID = "osx-x$Env:ARCH";
+}
+
+# On PowerShell 6 (Core) and later, Join-Path takes multiple child paths,
+# but we don't use that for compatability with Windows PowerShell (5.1 and
+# earlier).
+$TargetDirectory = (Join-Path (Join-Path $Env:PREFIX "opt") "iqsharp");
+$RepoRoot = Resolve-Path iqsharp;
+
+Write-Host "## Diagnostic Information ##"
+@{
+    "Path to .NET Core SDK" = $DotNetPath;
+    "Script root" = $PSScriptRoot;
+    "Prefix" = $Env:PREFIX;
+    "Runtime ID" = $RuntimeID;
+    "Target directory" = $TargetDirectory;
+    "Path to Jupyter" = (Get-Command jupyter -ErrorAction SilentlyContinue).Source;
+    "Repo root" = $RepoRoot;
+} | Format-Table | Out-String | Write-Host
+
+# We need to disable <PackAsTool>true</PackAsTool> when publishing
```

Comment:
Oh, I see. Can you just pass /property:PackAsTool=false from the command line, instead of having two versions of the .csproj?

Reply:
I tried that, and hit a bug with the publish step, such that this turned out to be more reliable. I hope to be able to remove this kludge with future .NET Core SDK versions, though.